Assets/Scripts/Blocks/RestoreableBlock.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RestoreableBlock : Block
+{
+    private Block_Data restoreableBlockData = new RestoreableBlock_Data();
+    public override Block_Data BlockData { get => restoreableBlockData; set => restoreableBlockData = value; }
+
+    [SerializeField]
+    private uint blockID = 0;
+
+    [SerializeField]
+    private Material restoredMaterial;
+
+    private Material restoreableMaterial;
+
+    #region Initialization / Destruction
+    public override void InitializeBlock(Block_Data data)
+    {
+        gameObject.GetComponent<Collider>().enabled = false;
```

Comment:
Use gameObject.GetComponent<Collider>().enabled = SceneManager.GetActiveScene().name != SceneDictionary.Play;
and remove the deleteBlockButton

Reply:
Ok